tg_bot.py

```diff
@@ -0,0 +1,47 @@
+from telethon import TelegramClient
+from telethon.tl.types import PeerChat, PeerChannel
+from telethon import utils
+import math
+from datetime import datetime, timezone
+
+# Remember to use your own values from my.telegram.org!
+api_id = 12345
+api_hash = '0123456789abcdef0123456789abcdef'
+client = TelegramClient('anon', api_id, api_hash)
+
+async def main():
+    # Getting information about yourself
+    me = await client.get_me()
+
+    # "me" is a user object. You can pretty-print
+    # any Telegram object with the "stringify" method:
+    #print(me.stringify())
+
+    # When you print something, you see a representation of it.
+    # You can access all attributes of Telegram objects with
+    # the dot operator. For example, to get the username:
+    username = me.username
+    #print(username)
+    #print(me.phone)
+    print("""
+    ##########################
+    ####### NEW RUN ##########
+    ##########################
+        """)
+
+    # You can print all the dialogs/conversations that you are part of:
+    chat_id = None
+    await client.get_dialogs()
+    async for dialog in client.iter_dialogs():
+        if dialog.name == "delete test group":
+            chat_id = dialog.id
```

Comment:
For performance purposes, there should be a break (not to continue iterating if group found)